src/applications/mhv-secure-messaging/tests/e2e/sort-filter-test/secure-messaging-add-filter-custom-date-custom-folder.cypress.spec.js

```diff
@@ -43,6 +43,8 @@ describe('SM CUSTOM FOLDER ADD FILTER CUSTOM DATE RANGE', () => {
 
   it(`verify errors`, () => {
     cy.get(Locators.BUTTONS.FILTER).click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(100);
```

Comment:
we should avoid adding implicit waits. Best practice would be to use some explicit wait in a way of finding some element that this wait relies on or some `.should()` assertion to wait